conda_forge_tick/mamba_solver.py

```diff
@@ -302,6 +302,30 @@ def _clean_reqs(reqs, names):
     return reqs
 
 
+def filter_problematic_reqs(reqs):
```

Comment:
This will sure break things that actually depend on this package. There has to be a better way and I skeptical we need this.

Reply:
nothing should depend on this anymore in this day and age, these are generally reserved for compatibility hacks with legacy states

Reply:
i'll fix this in another PR can we remove this code for now?

Reply:
We've chatted offline and are keeping this code for now.